lib/validation/jsonSchemas.js

```diff
@@ -350,6 +356,50 @@ schemas[validationKeys.TEST_LAUNCHER_INTERACTIVE] = {
 	},
 };
 
+schemas[validationKeys.TEST_LAUNCHER_TOKEN] = {
+	'type': 'object',
+	'required': ['token', 'presets'],
+	'properties': {
+		'token': {'type': 'string'},
+		'presets': {
+			'type': 'object',
+			'properties': {
+				'type': 'object',
+				'properties': {
+					'device': {
+						'oneOf': [
+							{
+								'type': 'string',
+							},
+							{
+								'type': 'object',
+								'required': ['deviceId'],
+								'properties': {
+									'deviceId': {'type': 'string'},
+								},
+							},
+						],
+					},
+					'config': {
+						'oneOf': [
+							{
+								'type': 'string',
+							},
+							{
+								'type': 'object',
+								'required': ['configId'],
+								'properties': {
```

Comment:
`config` object should also accept additional properties, maybe here need to use `additionalProperties`